### ISSUE TITLE
COURSEPLAY_IS_IN_TRAFFIC

### ISSUE THREAD

**Issue by knoxville73** [Courseplay/courseplay#3983]
Hello,
With the last version (245), the tractor is blocked without obstacle and move slowly on the road.
![245_1](https://user-images.githubusercontent.com/23388102/61576270-35931400-aad8-11e9-9612-add177276092.jpg)
![245_2](https://user-images.githubusercontent.com/23388102/61576273-3af05e80-aad8-11e9-9785-349461b63a0b.jpg)

![245](https://user-images.githubusercontent.com/23388102/61576195-1a73d480-aad7-11e9-856d-d452d63959de.jpg)

**Comment by schniedertoenz** [comment]
The same for me with #244. #242 is ok.

**Comment by Tensuko** [comment]
Can you press F5 ingame, and post a screenshot of that, to see the triggers?
I don't know what else is needed for the new collision detection.
@pvaiko ?

**Comment by Tensuko** [comment]
#3989

**Comment by pvaiko** [comment]
@Axel32019, this is yours.

**Comment by caplink811** [comment]
This behaviour affected mode2 too, harvester will never unload cause the unload driver mentioned about "blocked by obstacle" (the harvester which should unload). Also the speed on the field is reduced to a speed of 10 instead of the set speed of 32.

beside of all problems, I really appreciate this mod, thx.

**Comment by stangman04** [comment]
> This behaviour affected mode2 too, harvester will never unload cause the unload driver mentioned about "blocked by obstacle" (the harvester which should unload). Also the speed on the field is reduced to a speed of 10 instead of the set speed of 32.
> 
> beside of all problems, I really appreciate this mod, thx.

I have noticed this as well. Just updated to v245.

**Comment by Tensuko** [comment]
Can you test this with 262 or higher and report ?

**Comment by stangman04** [comment]
My pc is apart due to hardware issue but last I checked this was resolved

**Comment by Tensuko** [comment]
Closed. If anything new make new Issue.
